validate_package_xml/action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Validate package.xml'
+description: 'Validate package.xml file'
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: package.xml syntax validation
+      shell: bash
+      run: |
+        echo "Install ament-xmllint"
+        sudo apt-get install -y python3-venv libxml2-utils
+        python3 -m venv .venv
+        source .venv/bin/activate
+        pip install ament-xmllint
+        ament_xmllint package.xml 2>&1 | tee $GITHUB_STEP_SUMMARY
```

Comment:
per https://github.com/gazebosim/gz-cmake/pull/413#issuecomment-2045933120, probably this whole step so people aren't mistaken. that it is running when it's actually not

Reply:
"probably this whole step"? Is there a word missing?

Reply:
oops; I meant we should comment out the whole step (as opposed to just the line invoking `ament_xmllint`